content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -144,13 +144,77 @@ For more information on version skews, see:
 * Kubernetes [version and version-skew policy](/docs/setup/release/version-skew-policy/)
 * Kubeadm-specific [version skew policy](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#version-skew-policy)
 
+{{< note >}}
+Kubernetes has two different package repositories starting from August 2023.
+The Google-hosted repository is deprecated and it's being replaced with the
+Kubernetes (community-owned) package repositories. The Kubernetes project strongly
+recommends using the Kubernetes community-owned package repositories, because the
+project plans to stop publishing packages to the Google-hosted repository in the future.
+
+There are some important considerations for the Kubernetes package repositories:
+
+- The Kubernetes package repositories contain packages beginning with those
+  Kubernetes versions that were still under support when the community took
+  over the package builds. This means that anything before v1.24.0 will only be
+  available in the Google-hosted repository.
+- There's a dedicated package repository for each Kubernetes minor version.
+  When upgrading to to a different minor release, you must bear in mind that
+  the package repository details also change.
+
+To learn more, you can read the official announcement ["pkgs.k8s.io: Introducing
+Kubernetes community-owned package repositories"](/blog/2023/08/15/pkgs-k8s-io-introduction/).
```

Comment:
As it seems the blog has been scheduled for 22 August
```suggestion
Kubernetes community-owned package repositories"](/blog/2023/08/22/pkgs-k8s-io-introduction/).
```

Reply:
I think it makes sense to publish that on the 15th, and I'll see if release comms agree.